index.html

```diff
@@ -78,196 +82,192 @@ <h3 class="contact-p">LET’S CONNECT</h3>
         </div>
       </section>
       <section id="works">
-        <div class="wraper">
-          <div class="image">
-            <img alt="work1" src="./assets/imgs/Snapshoot1.svg" />
-          </div>
-          <div class="work-info">
-            <h2 class="work-child">Tonic</h2>
-            <div class="description-wraper work-child">
-              <h3 class="project-name">CANOPY</h3>
-              <img alt="dot" src="./assets/imgs/Counter.png" />
-              <p>Back End Dev</p>
-              <img alt="dot" src="./assets/imgs/Counter.png" />
-              <p>2015</p>
+        <div class="works-container">
+          <div class="wraper">
+            <div class="image card1"></div>
+            <div class="work-info">
+              <h2 class="work-child">Tonic</h2>
+              <div class="description-wraper work-child">
+                <h3 class="project-name">CANOPY</h3>
+                <img alt="dot" src="./assets/imgs/Counter.png" />
+                <p>Back End Dev</p>
+                <img alt="dot" src="./assets/imgs/Counter.png" />
+                <p>2015</p>
+              </div>
+              <p class="work-child">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="frameworks-list work-child">
+                <li class="framework">html</li>
+                <li class="framework">css</li>
+                <li class="framework">javaScript</li>
+              </ul>
+              <button type="button" class="project-btn work-child">
+                See Project
+              </button>
```

Comment:
- [ ] Based on project requirements,  see project `button` should take users to the Live Demo of the project or to the GitHub repository. Please add these links for all  4 see project `buttons` 👍🏻